starfish/spacetx_format/schema/experiment_4.0.0.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$id": "https://github.com/spacetx/starfish/blob/0.0.22/validate_sptx/schema/experiment.json",
```

Comment:
I thought you were going to point this at the commit prior to 5.0.0

Reply:
i.e. just 0.0.21 or the actual commit? Sorry. It paged out.

Reply:
Any commit that references the 4.0.0 schema should be ok.

Reply:
Ok. The version variable was updated in 16ecf10da06dbf3e4bae7ad67e4f43419383e4ed, so using its parent: d933c08a6987da26433714f23cf331e067087bf6

Reply:
(Note: that commit is somewhere between 0.0.26 and 0.0.27)